coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -4445,19 +4445,67 @@ def forward(self, x, y):
 
         if reverse_input_order:
             input_output_strings = equation.split("->")
-            input_strings = input_output_strings[0].split(",")
-            equation = (
-                input_strings[1]
-                + ","
-                + input_strings[0]
-                + "->"
-                + input_output_strings[1]
-            )
-            input_shapes = [input_shapes[1], input_shapes[0]]
+            input_string = ",".join(reversed(input_output_strings[0].split(",")))
+            equation = input_string + "->" + input_output_strings[1]
+            input_shapes.reverse()
             if converter_input_type is not None:
-                converter_input_type = [converter_input_type[1], converter_input_type[0]]
+                converter_input_type.reverse()
+
+        model = TestBinaryEinsum()
+        self.run_compare_torch(
+            input_shapes,
+            model,
+            backend=backend,
+            compute_unit=compute_unit,
+            input_as_shape=True,
+            converter_input_type=converter_input_type
+        )
 
-        model = TestEinsum()
+    @pytest.mark.parametrize(
+        "compute_unit, backend, equation, dynamic",
+        itertools.product(
+            compute_units,
+            backends,
+            ["ab->ba", "aa->a", "ab->b", "iijk->ji"],
+            [False, True],
+        ),
+    )
+    def test_unary_einsum(self, compute_unit, backend, equation, dynamic):
+        class TestUnaryEinsum(nn.Module):
+            def forward(self, x):
+                return torch.einsum(equation, x)
+        if backend == ("mlprogram", "fp16") and dynamic:
+            pytest.xfail("rdar://106631543 ([Infra]Re-enable the unittests for torch einsum ops)")
```

Comment:
I'm guessing this is a copy and paste mistake. I don't think we should have this xfail. Certainly the radar id is not valid. Do we need this?